Frameworks/CoreFoundation/Base.subproj/CFSystemDirectories.c

```diff
@@ -55,7 +55,8 @@ CFSearchPathEnumerationState __CFGetNextSearchPathEnumeration(CFSearchPathEnumer
 #endif
 
 
-#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_WINDOWS
+// WINOBJC: Disable this function only for WinObjC as we don't provide system intrinsics for it.
```

Comment:
This was never in the .def, so it was being stripped out anyway.
The native functions I'm referring to are above, on 38 and 42. _"We use the System framework implementation on Mach."_

---
In reply to: [72318824](https://github.com/Microsoft/WinObjC/pull/684#discussion_r72318824) [](ancestors = 72318824)